docs/components/links/README.md

```diff
@@ -278,14 +278,17 @@ This component has compliance with following WebAIM’s accessibility guidelines
 
 ## Don't Use When
 
-- Navigating a user from promotional or campaign content. Instead, use [Call to Action](../cta/)
+- Triggering interface interactions, such as opening an modal or progressing a user through a flow. Instead, use [Buttons](../buttons/)
```

Comment:
i think we should remove this line, as we do support passing `tag="button" into links